docs/cloud/introduction.md

```diff
@@ -59,14 +57,24 @@ and click the button to resend it.
 
 ## Node-RED on FlowFuse Cloud
 
-FlowFuse currently offers Node-RED 2.2 and 3.0 to customers. When creating a
-new instance a [Stack](/docs/user/concepts.md#stack) is chosen, which later
+FlowFuse currently offers Node-RED 4.x, 3.x and 2.x to customers. When creating a
```

Comment:
```suggestion
FlowFuse currently offers Node-RED 4.x, 3.x and 2.x to customers. When creating a
```

We really should drop 2.x 

And we offer 3.0.x and 3.1.x builds

Reply:
I think its technically sufficiently correct for this update.